hs-init.hs

```diff
@@ -508,7 +514,11 @@ customizeLicense l t nm year
 -- This is needed to be able to call commands by writing strings.
 instance (a ~ Text, b ~ ()) => IsString ([a] -> IO b) where
   fromString "cd" [arg] = setCurrentDirectory $ T.unpack arg
-  fromString cmd args   = callCommand $ showCommandForUser cmd (map T.unpack args)
+  fromString cmd args   = callProcess cmd (map T.unpack args)
+
+deleteFile :: FilePath -> IO  ()
+deleteFile file = catch (removeFile file) printError
```

Comment:
I like this solution! Great! I think we can continue and this trash file that didn't removed is not the reason to stop the process

Reply:
In case there is confusion, this function does allow the process to continue simply printing a message if the delete fails, but not stopping the process.

Reply:
Yes, exactly what is needed! :+1: